apps/frontend/src/api-util.ts

```diff
@@ -5,6 +5,7 @@ export type GetListOptions = {
   filters: Set<string>;
   currentPage: number | null;
   pageSize: number | null;
+  teamId?: string;
```

Comment:
Not really thrilled by this. This should be moved into a separate type that extends this one specifically for research outputs as teamId won't apply to other endpoints.